tox.ini

```diff
@@ -54,14 +54,17 @@ passenv=GIT_COMMIT
 sitepackages=true
 allowlist_externals = git
 deps =
-    -rrequirements.txt
-    -rrequirements-dev.txt
+    -r requirements.txt
+    -r requirements-dev.txt
 commands =
     ; the GIT_COMMIT env is the current commit of the core repo
     ; NOTE!!!! We must clean this up in the jenkins configuration with `rm -rf "nvtabular-$GIT_COMMIT"`
     git clone --depth 1 https://github.com/NVIDIA-Merlin/models.git models-{env:GIT_COMMIT}
     python -m pip install --upgrade ./models-{env:GIT_COMMIT}[dev,implicit,lightfm,tensorflow]
-    python -m pip install --upgrade git+https://github.com/NVIDIA-Merlin/NVTabular.git
+    git clone --depth 1 https://github.com/NVIDIA-Merlin/NVTabular.git NVTabular-{env:GIT_COMMIT}
+    pip install --upgrade ./NVTabular-{env:GIT_COMMIT}
+    pip install -r .NVTabular-{env:GIT_COMMIT}/requirements/test.txt
```

Comment:
We shouldn't be installing additional requirements files like this. They should either be listed in the `deps` section, or provided as a param on the install like `pip install nvtabular[test]`

Reply:
fixed